app/src/test/java/edu/uc/haflga/financialpal/ExampleUnitTest.kt

```diff
@@ -10,7 +10,7 @@ import org.junit.Test
  */
 class ExampleUnitTest {
     @Test
-    fun addition_isCorrect() {
+    fun additionIsCorrect() {
```

Comment:
This is OK as it was... test functions have a different naming convention than other functions.